contrib/epee/include/net/net_ssl.h

```diff
@@ -146,8 +147,8 @@ namespace net_utils
 	bool create_ec_ssl_certificate(EVP_PKEY *&pkey, X509 *&cert);
 	bool create_rsa_ssl_certificate(EVP_PKEY *&pkey, X509 *&cert);
 
-  //! Store private key for `ssl` at `base + ".key"` unencrypted and certificate for `ssl` at `base + ".crt"`.
-  boost::system::error_code store_ssl_keys(boost::asio::ssl::context& ssl, const boost::filesystem::path& base);
+  //! Extract private key and certificate from ssl context `ssl` and store into respective file locations, unencrypted
```

Comment:
Now one can give specific paths for the private key and certificate. `core_rpc_server` still uses the current paths: `rpc_ssl.key` and `rpc_ssl.crt` in the data directory.